library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
```

Comment:
The name create_window confused me before I got to this point. I'd say that creation of the structure should be separated from drawing.